onnxruntime/contrib_ops/cuda/bert/skip_layer_norm_impl.cu

```diff
@@ -102,14 +105,19 @@ bool LaunchSkipLayerNormKernel(
     const void* skip,
     const void* gamma,
     const void* beta,
-    const void* bias,
+    const void* bias,    
+    float epsilon,
     int hidden_size,
     int element_count,
     size_t element_size) {
   // use default stream
   const cudaStream_t stream = nullptr;
 
   if (element_size == 2) {
+    const half epsilon_half = __float2half_rn(epsilon);
+    if (__heq(epsilon_half, __int2half_rn(0))) {
```

Comment:
why do we need this warning? For skipLayerNorm with default epsilon, it will always be 0, right? And it prints warning for each call. It will hurt the performance a lot.